### PR TITLE
Added the possibility to choose exportation format of the Participation (default .xls)

### PR DESCRIPTION
**Pull request by yanicolivier**
| Q                                                      | R
| ------------------------------------------ | -------------------------------------------
| Type of contribution ?                      | [Enhancement]
| Tickets (_issues_) concerned               | [[72](https://genielibre.com/issues/72)]

---

##### What have you changed ?
Change the exportation from .csv to .xls

##### Verification :

**PR standard**
 - [X] Branches naming convention: 
     - `prefix-snake_case_description`
     - ex: `enhancement-create_new_cell`
     - ex: `fix-pep8_standard_on_cell_model`
 - [X] Fill in this automatically generated PR template
 - [X] Do not include issue numbers in the PR title
 - [X] Include screenshots and animated GIFs in your pull request whenever possible

**Code standard**
 - [X] This Pull-Request fully meets the requirements defined in the issue
 - [] Add or modify the attached tests
 - [X] Follow the Python Styleguide
 - [X] Document new code based on the Documentation Styleguide
 - [X] Use I18N functions when you add some text

### CODE REVIEW

File: source/apiVolontaria/volunteer/tests/tests_admin_participation_export.py [hunk 1]
```diff
@@ -129,6 +129,9 @@ def test_export(self):
         self.assertNotContains(
             response, self.participation2.user.email, status_code=200
         )
+
+        print(response.content)
```
Comment (RignonNoel) [inline source/apiVolontaria/volunteer/tests/tests_admin_participation_export.py]: Remove this line please

**Comment by yanicolivier** [comment]
@RignonNoel I agree to keep both options.  What about putting in place 2 routes, something likes this :
_r'^cells/(?P<pk>\d+)/export-csv$',_
_r'^cells/(?P<pk>\d+)/export-xls$',_

**Comment by RignonNoel** [comment]
@yanicolivier Why don't we use a query params like for the filters you created ? So we keep just one route, we don't create a breaking change, and a new option is available :)

**Comment by yanicolivier** [comment]
@RignonNoel Perfect !

**Pull request closed without merging**